php/README.md

```diff
@@ -0,0 +1,24 @@
+# HubSpot-Php sample OAuth 2.0 app
+
+### Setup App
+
+Make sure you have [Php 7.0+](https://www.php.net/downloads) and [Composer](https://getcomposer.org) installed.
```

Comment:
Can we mention about required php extensions (php_curl) here?

Reply:
It works correctly without php_curl